test/api_integration/apis/status/status.js

```diff
@@ -39,10 +39,6 @@ export default function ({ getService }) {
           expect(body.status.overall.state).to.be('green');
 
           expect(body.status.statuses).to.be.an('array');
-          const kibanaPlugin = body.status.statuses.find((s) => {
-            return s.id.indexOf('plugin:kibana') === 0;
-          });
-          expect(kibanaPlugin.state).to.be('green');
```

Comment:
Removing the `init` function from the legacy kibana plugin removed the plugin entry from the status, so I had to adapt two tests relying on that. As we are very close to remove the plugin totally, I felt like it was better to remove that than to preserve an empty `init` function.

It's also likely we'll adapt and enhance these tests once we finish the status API migration anyway.